governance/polls/Ratification Poll - Declaration of Intent - Invest in Short-Term Bonds (MIP13c3-SP12) - March 14, 2022.md

```diff
@@ -5,6 +5,7 @@ discussion_link: https://forum.makerdao.com/t/mip13c3-sp12-declaration-of-intent
 vote_type: Plurality Voting
 categories:
    - MIPs
+   - Impact-Medium
```

Comment:
Don't change the old polls, it breaks the hash. Can you take this out?

Reply:
Sorry. My bad. Done!